### PR TITLE
Adding an example that shows MirroredStrategy

### PR DESCRIPTION
**Pull request by sayakpaul**
The example already does a great job discussing different flavors of distributed training for Keras models. In the **Further reading** section, I just added a link to my article that shows the use of `MirroredStrategy` in a bit more detailed way.

### CODE REVIEW

File: guides/distributed_training.py [hunk 1]
```diff
@@ -456,4 +456,5 @@ def run_training(epochs=1):
     https://www.tensorflow.org/api_docs/python/tf/distribute/MirroredStrategy)
 4. [MultiWorkerMirroredStrategy docs](
     https://www.tensorflow.org/api_docs/python/tf/distribute/experimental/MultiWorkerMirroredStrategy)
+5. [Distributed training in `tf.keras`](https://towardsdatascience.com/distributed-training-in-tf-keras-with-w-b-ccf021f9322e)
```
Comment (fchollet) [inline guides/distributed_training.py:459]: Thank you for the PR. Please use the full title of the article here. The link text should differentiate this link from the others, and should answer: why should the reader click on it?

Comment (sayakpaul) [reply guides/distributed_training.py]: Thanks for the suggestion, I updated it. The article shows the use of `MirroredStrategy` and also discusses about some important gotchas like learning rate tuning, data pipeline optimization. Should I change the title of the article to reflect on that?
